src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,124 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+import duke.task.Todo;
+import duke.task.Event;
+import duke.task.Deadline;
+import duke.task.TaskList;
+
+/**
+ * Represents a storage space for tasks on hard-drive.
+ * It handles the loading of tasks from a file and saving of tasks to the same file.
+ *
+ * @author  Elumalai Oviya Dharshini
+ * @version 0.1
+ */
+public class Storage {
+    protected String filePath;
+    ArrayList<Task> tasks;
+
+    /**
+     * Suppresses any unused warnings from a given boolean result.
+     *
+     * @param result variable to suppress warnings from
+     */
+    //@@author Eric Lange-reused
+    //Reused from https://stackoverflow.com/questions/27904329/warning-file-mkdir-is-ignored#answer-54341862
+    // with minor modifications
+    @SuppressWarnings("unused")
+    private static void IGNORE_RESULT(boolean result) {
```

Comment:
I am not sure what does this means, it is not returning anything

Reply:
This utility function is defined for ```mkdirs``` - which makes parent directories of a given location and returns ```True``` or ```False``` depending on whether directories were created. This suppresses the ```File.mkdirs() is ignored``` warning that results from not handling the return value of ```mkdirs```. 

I am not certain if this is the recommended way to handle it so I am leaving it as is for now.